### ISSUE TITLE
Inference CoverNet

### ISSUE THREAD

**Issue by MaxLyubimov** [nutonomy/nuscenes-devkit#346]
Hello. Can you explain how I can get trajectory using  CoverNet output layer? For example MPT returns points for each mode. I want to get the same from CoverNet. Thank you  for answer !

**Comment by nachiket92** [comment]
Thank you for releasing the prediction benchmark and for sharing the code for CoverNet.
Related to the original question raised by @MaxLyubimov, would it be possible to share the code for trajectory generation (Sec 3.3 and 3.4 of the paper)? This should allow for inference, from what I understand.  Alternatively, it would also be helpful if the trajectory sets could be shared. Thanks again!

**Comment by alexbuyval** [comment]
I have the same question. Thanks!

**Comment by freddyaboulton** [comment]
Hi @MaxLyubimov @nachiket92 @alexbuyval -

Sorry for the delay! We uploaded the fixed trajectory sets for epsilon=2, 4, 8 [here](https://www.nuscenes.org/public/nuscenes-prediction-challenge-trajectory-sets.zip). The kth index of the CoverNet output corresponds to the kth trajectory in the trajectory set.

I opened a PR to add this to the prediction tutorial (https://github.com/nutonomy/nuscenes-devkit/pull/353)
